app/models/solr_document.rb

```diff
@@ -359,8 +359,12 @@ def playlist_group
   def playlist_order
     self[Solrizer.solr_name('playlist_order')]
   end
-  
+
   def media_src(part)
     "/media/#{id}?part=#{part.to_s}"
   end
+
+  def digitized?
+    !sonyci_id.nil? ? sonyci_id.any? : false
+  end
```

Comment:
this is fine, but you can also do bang bang... i.e. `!!sonyci_id`. This will cast anything to it's boolean equiv. Just gotta be careful with 0, which will cast to true.

Reply:
Oh... nevermind.. i guess `sonyci_id` is either nil, an array of stuff, or an empty array?